benchmarks/nel/README.md

```diff
@@ -47,9 +49,18 @@ in the project directory.
 
 | File | Source | Description |
 | --- | --- | --- |
-| `assets/reddit.zip` | URL | Entity linking dataset scraped from Reddit. See [paper](https://arxiv.org/abs/2101.01228). |
+| `assets/wiki/wikidata_entity_dump.json.bz2` | URL | Wikidata entity dump. Download can take a long time! |
+| `assets/wiki/wikipedia_dump.xml.bz2` | URL | Wikipedia dump. Download can take a long time! |
+| `assets/wiki/wikidata_entity_dump_filtered.json.bz2` | URL | Filtered Wikidata entity dump for demo purposes. |
+| `assets/wiki/wikipedia_dump_filtered.xml.bz2` | URL | Filtered Wikipedia dump for demo purposes. |
 
 <!-- SPACY PROJECT: AUTO-GENERATED DOCS END (do not remove) -->
 
-Note that `svn` is required for downloading the Mewsli-9 dataset (should be replaceable by `git` with sparse 
-checkout).
\ No newline at end of file
+Notes: 
```

Comment:
There should be a very clear & visible warning that this currently doesn't work on Windows

Reply:
Added one. If you think it's insufficient/would like to change something, feel free to reopen the conversation.